core/src/toga/app.py

```diff
@@ -1020,6 +1020,16 @@ def set_full_screen(self, *windows: Window) -> None:
     # End backwards compatibility
     ######################################################################
 
+    def focus(self):
+        """Bring the application into focus.
+
+        This method will attempt to bring the application window into focus. Note that
+        it is generally considered poor practice for applications to steal focus from
+        the user, so use this method sparingly.
+
+        This is a no-op on mobile and console platforms.
```

Comment:
We try not to call out specific platforms in docstrings; we put platform specific notes in the [Notes section](https://toga.readthedocs.io/en/stable/reference/api/app.html#notes) for the class in question.

```suggestion
        """Force the app to be the currently active app on the user's desktop.

        This request will not be honoured by all platforms; see the :ref:`platform notes
        <app-notes>` for details.

        This method should be used sparingly, if at all. It is generally considered poor
        practice for an application to steal focus from other applications.
```

(This will also require adding a `.. app-notes:` declaration just before the Notes header in the app documentation.)